closed/autoconf/custom-spec.gmk.in

```diff
@@ -57,6 +57,9 @@ endif
 # DDR
 OPENJ9_ENABLE_DDR       := @OPENJ9_ENABLE_DDR@
 
+## JITSERVER
+OPENJ9_ENABLE_JITSERVER    := @OPENJ9_ENABLE_JITSERVER@
```

Comment:
Can you please remove the extra `#` here, and remove the extra spaces before `:=`.